src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -447,7 +447,7 @@ def load_arguments(self, _):
     with self.argument_context('webapp vnet-integration') as c:
         c.argument('name', arg_type=webapp_name_arg_type, id_part=None)
         c.argument('slot', help="the name of the slot. Default to the productions slot if not specified")
-        c.argument('vnet', help="Vnet name", validator=validate_add_vnet)
```

Comment:
if this validation isn't used anywhere else remove the validation itself from validators.py

Reply:
Same function is being used by **functionapp vnet-integration**.